tools/display_image.php

```diff
@@ -20,7 +20,7 @@
     $error_message = $exception->getMessage();
 }
 
-$title = _("Display Image for page");
+$title = sprintf(_("Display Image: %s"), $imagefile);
```

Comment:
`$imagefile` may be blank here. If so we should just have the title be "Display Image" or something similar.

Reply:
Is title encoded by the header function? If not may want to encode image here

Reply:
Just checked, it is encoded: https://github.com/DistributedProofreaders/dproofreaders/blob/master/pinc/html_page_common.inc#L58 and

Reply:
Changed the way this is done.